src/cpu/x64/jit_avx512_common_conv_kernel.cpp

```diff
@@ -822,6 +822,13 @@ status_t jit_avx512_common_conv_fwd_kernel::init_conf(jit_conv_conf_t &jcp,
     jcp.stride_h = (ndims == 3) ? 1 : cd.strides[ndims - 4];
     jcp.stride_w = cd.strides[ndims - 3];
 
+    // Big int (> INT_MAX) values are unsupported and jcp fields may overflow
+    // TODO: change data type of jcp fields to size_t
+    if ((ndims == 5 && cd.dilates[0] > INT_MAX)
```

Comment:
For consistent style.
```suggestion
    if ((ndims == 5 && cd.dilates[ndims - 5] > INT_MAX)
```

Reply:
done, thanks